catalog/core/catalog-core-sourcemetricsplugin/src/main/java/ddf/catalog/metrics/source/SourceMetricsImpl.java

```diff
@@ -172,414 +73,55 @@ public QueryRequest process(Source source, QueryRequest input)
   public QueryResponse process(QueryResponse input)
       throws PluginExecutionException, StopProcessingException {
 
-    LOGGER.trace("ENTERING: process (for PostFederatedQueryPlugin)");
-
-    if (null != input) {
+    if (input != null) {
```

Comment:
✏️ Reverse the predicate, return null early and collapse the larger nested `if`?

```
if (input == null ) {
     LOGGER.trace("EXITING: process (for PostFederatedQueryPlugin)");

    return null;
}
....
```